src/nsidc/metgen/metgen.py

```diff
@@ -142,12 +142,6 @@ def scrub_json_files(path):
         except Exception as e:
             print('Failed to delete %s: %s' % (file_path, e))
 
-def fn_process(configuration):
```

Comment:
Haha - I assumed this was doing some sort of magic with the new pipeline and didn't ask about it in previous reviews.

Reply:
Heh, yeah I saw that I left it in there and hoped you wouldn't see this. 😆